drivers/drivers_test.go

```diff
@@ -26,14 +26,28 @@ func TestS3URL(t *testing.T) {
 
 func TestCustomS3URL(t *testing.T) {
 	assert := assert.New(t)
-	os, err := ParseOSURL("s3+http://user:password@example.com:9000/path/to/bucket-name", true)
+	os, err := ParseOSURL("s3+http://user:password@example.com:9000/bucket-name", true)
 	s3, iss3 := os.(*s3OS)
 	assert.Equal(true, iss3)
 	assert.Equal(nil, err)
 	assert.Equal("http://example.com:9000", s3.host)
 	assert.Equal("bucket-name", s3.bucket)
 	assert.Equal("user", s3.awsAccessKeyID)
 	assert.Equal("password", s3.awsSecretAccessKey)
+	assert.Equal("ignored", s3.region)
+}
+
+func TestCustomS3URLWithRegion(t *testing.T) {
+	assert := assert.New(t)
+	os, err := ParseOSURL("s3+http://user:password@example.com:9000/path/to/region-name/bucket-name", true)
```

Comment:
What happened to `/path/to`? There's no `assert.Equal` that has it.

Reply:
It is ignored. It was always ignored (`hosturl.Path = ""`)